CONTRIBUTING.txt

```diff
@@ -330,6 +330,12 @@ the documentation.
 .. code:: sh
 
     pip install sphinx
+    pip install sphinx_gallery
+    pip install sphinx-copybutton
+
+`copybutton file <https://github.com/choldgraf/sphinx-copybutton/blob/master/sphinx_copybutton/_static/copybutton.css>`__ is modified to generate the online documentation by
+adding ``background: transparent no-repeat none; border: none;`` and
+adjusting ``margin: 0.5px; height: 1.6em; padding: 0 2px 0px;`` at ``a.copybtn``.
```

Comment:
I don't understand why this comment is here?

Reply:
@jni  I am also really glad to see the click button. Thanks for the suggestion. I have to use the custom css  ( modified CSS in sphinx-copybutton) to generate a nicer and transparent click-button because the button blocks some text in a few cases. Also I am not sure how the online document is generated. Should I point the forked repository with the custom css file for sphinx-copybutton installation.